.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+jobs:
```

Comment:
I vote to leave with the workflow config even though we have just one step now. There is an difference on configs with and without workflows, so for future extensions that is important

Reply:
Done!